src/schema/resolvers/user.js

```diff
@@ -0,0 +1,14 @@
+import * as userController from '~/src/controllers/user'
+
+const resolvers = {
+  Query: {
+    currentUser: userController.getCurrentUser
+  },
+  Mutation: {
+    signUp: userController.signUp,
```

Comment:
Let's rename this to "signup" (all lowercase).

Reply:
Done